### PR TITLE
container: cleanup container systemd units

### PR DESCRIPTION
**Pull request by clwluvw**

### CODE REVIEW

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

**Comment by clwluvw** [comment]
jenkins test centos-container-all_daemons

File: tox-cephadm.ini [hunk 1]
```diff
@@ -32,7 +32,7 @@ commands=
   bash {toxinidir}/tests/scripts/vagrant_up.sh --no-provision {posargs:--provider=virtualbox}
   bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
 
-  ansible-playbook -vv -i {changedir}/hosts {toxinidir}/infrastructure-playbooks/cephadm.yml --extra-vars "\
+  ansible-playbook -vv --diff -i {changedir}/hosts {toxinidir}/infrastructure-playbooks/cephadm.yml --extra-vars "\
```
Comment (guits) [inline tox-cephadm.ini:35]: a commit message (even very brief) to explain why would be nice.
(think about people who don't have any context ☺️ )

**Comment by guits** [comment]
jenkins test centos-container-lvm_osds

**Review by guits — APPROVED** [review]

**Pull request merged**